### DISCUSSION TITLE
configuring WiFI

### DISCUSSION THREAD

**Discussion by wkraaij** [rvdbreemen/OTGW-firmware#80] [Q&A]
I just bought the Nohoshop OTGW module wit a WeMos module. I cannot configure the WiFi, since the WeMos will not go in AP mode. Is there another way to configure the WiFi or do I need to perform a reset?

**Comment by DaveDavenport** [comment]
Did you flash the latest version of the firmware onto the wemos? (if it is a wemos you used in other projects, it is good to clear the full flash before flashing the firmware).

https://github.com/rvdbreemen/OTGW-firmware/wiki/How-to-flash-the-binary-release-to-your-ESP8266
